packages/circuit-ui/components/RadioButton/RadioButton.spec.tsx

```diff
@@ -15,83 +15,95 @@
 
 import { createRef } from 'react';
 
-import {
-  create,
-  renderToHtml,
-  axe,
-  render,
-  userEvent,
-} from '../../util/test-utils';
+import { render, userEvent, axe } from '../../util/test-utils';
```

Comment:
Most of the diff in this spec is cleanup:

- replaced `create` and `renderToHtml` by `render` (repo standards)
- wrapped tests in describe blocks "Styles", "Logic" and "Accessibility" (repo standards)

The changes are isolated in separate commits, which can make reviewing easier.